app/partials/content.html

```diff
@@ -1,15 +1,65 @@
-<div class="row marketing">
-  <div class="col-lg-6">
-    <h4>HTML5 Boilerplate</h4>
-    <p>HTML5 Boilerplate is a professional front-end template for building fast, robust, and adaptable web apps or sites.</p>
-
-    <h4>Sass</h4>
-    <p>Sass is the most mature, stable, and powerful professional grade CSS extension language in the world.</p>
+<br><br>
+<div class="row" id="products">
+    <div class="col-lg-2 col-md-offset-1">
+    <img src="partials/images/prod1.png"/></div>
+    <div class="col-lg-1">
+    <p>BRANDED</p>
+    <p>SHOES</p>
+    <p>SHOP</p></div>
+    <div class="col-lg-2 col-md-offset-1">
+    <img src="partials/images/prod2.png"/></div>
+    <div class="col-lg-1">
+    <p>BRANDED</p>
+    <p>TSHIRTS</p>
+    <p>SHOP</p></div>
+    <div class="col-lg-2">
+    <img src="partials/images/prod3.png"/></div>
+    <div class="col-lg-1">
+    <p>BRANDED</p>
+    <p>SHOES</p>
+    <p>SHOP</p></div>
+  </div>
+<br><br>
 
-    <h4>Bootstrap</h4>
-    <p>Sleek, intuitive, and powerful mobile first front-end framework for faster and easier web development.</p>
-    <h4>Modernizr</h4>
-    <p>Modernizr is an open-source JavaScript library that helps you build the next generation of HTML5 and CSS3-powered websites.</p>
+  <div id="featured">
+  <br>
+    <h3>FEATURED PRODUCTS</h3>
+  <br>
+  </div>
 
+<div class="row" id="other">
+    <div class="col-lg-2 col-md-offset-2">
+    <img src="partials/images/prod4.png"/>
+    <p>BRANDED TEES</p>
+    <p>$300</p>
+    <p>BUY NOW</p></div>
+    <div class="col-lg-2 col-md-offset-1">
+    <img src="partials/images/prod5.png"/>
+    <p>BRANDED JACKET</p>
+    <p>$300</p>
+    <p>BUY NOW</p></div>
+    <div class="col-lg-2 col-md-offset-1">
+    <img src="partials/images/prod6.png"/>
+    <p>BRANDED SHOE</p>
+    <p>$300</p>
+    <p>BUY NOW</p></div>
+  </div>
+<br><br>
+  <div class="row" id="other">
```

Comment:
You can't have 2 or more elements with the same id, other was already used somewhere else.